src/schema/types/film.ts

```diff
@@ -30,7 +30,7 @@ import VehicleType from './vehicle';
 /**
  * The GraphQL type equivalent of the Film resource
  */
-const FilmType = new GraphQLObjectType({
+const FilmType: GraphQLObjectType = new GraphQLObjectType({
```

Comment:
for some reason if I remove the type from this assignment. I get. 


```typescript
const FilmType: any
The GraphQL type equivalent of the Film resource

'FilmType' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.ts(7022)```